resources/known-amendments.md

```diff
@@ -360,6 +223,28 @@ This amendment was intended to add support for several types of crypto-condition
 However, the amendment was added to `rippled` v0.60.0 before implementation was complete. As a result, this amendment ID refers to incomplete code which does almost nothing. Modifying the existing amendment to add support for other crypto-conditions would cause a conflict with old versions of the amendment already in released software. If a future release adds support for additional crypto-conditions, it must use a new and different amendment ID.
 
 
+### DID
```

Comment:
Decentralized Identifier comes before DeletableAccounts, but DID doesn't. Should the heading be Decentralized Identifier?

Reply:
No, the heading is the name of the amendment, hence will leave it as DID, or AMM and so on.